### ISSUE TITLE
Add value/initialValue?

### ISSUE THREAD

**Issue by gabrielhpugliese** [ubilabs/react-geosuggest#13]
Does it make sense?

**Comment by ro-ka** [comment]
You can already pass in values. [See the readme](https://github.com/ubilabs/react-geosuggest#fixtures). What else do you mean?

**Comment by plraphael** [comment]
Here is what I think he meant:
Suppose we have a form and whats to get the address in this form with Geosuggest and suppose this form also performs uptades. This means that we would need to initialize Geosuggest with the address we got from it already when the user filled in the form to insert...
Now, the question: is there a way for us to initialize the input of Geosuggest right now? If there is, I think it is not in the docs...

I have modified the code of your Geosuggest for my need, to include a prop named "defaultValue" and the new getInitialState goes something like this:
```js
/**
   * Get the initial state
   * @return {Object} The state
   */
  getInitialState: function() {
    return {
      isSuggestsHidden: true,
      userInput: this.props.defaultValue,
      activeSuggest: null,
      suggests: [],
      geocoder: new google.maps.Geocoder(),
      autocompleteService: new google.maps.places.AutocompleteService()
    };
  },
```
What do you think ro-ka? Am I missing something terrible here? :)

Thanks

**Comment by ro-ka** [comment]
Thanks for the feedback –sounds useful! Will think about the implementation and look at it next week. Planning to include this.

**Comment by plraphael** [comment]
That is nice @ro-ka, thanks :)

**Comment by ro-ka** [comment]
Released in `1.4.0`.
